modules/hotp-verification

```diff
@@ -2,12 +2,12 @@ modules-$(CONFIG_HOTPKEY) += hotp-verification
 
 hotp-verification_depends := libusb $(musl_dep)
 
-# v1.6
-hotp-verification_version := e9050e0c914e7a8ffef5d1c82a014e0e2bf79346
+# v1.6 + patch from https://github.com/Nitrokey/nitrokey-hotp-verification/pull/46/commits/de355ed93ba50280bf377772082b76b7a2285185
+hotp-verification_version := de355ed93ba50280bf377772082b76b7a2285185
```

Comment:
This is https://github.com/Nitrokey/nitrokey-hotp-verification/pull/46#issue-2691319202 original PR state commit, not implementing argument for single SECRET_APP_PIN to be set since no default PIN set as gpg counterpart.

This is commit https://github.com/Nitrokey/nitrokey-hotp-verification/pull/46/commits/de355ed93ba50280bf377772082b76b7a2285185 for the curious eyes.

Reply:
now using 1.6 commit + patches per https://github.com/linuxboot/heads/pull/1850/commits/a9d3d96ec11452eb48cb360894c28ad029f6e38b

only testing PR 46 because conflits with PR 43.